.happy/terraform/modules/deletion/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable stack_resource_prefix {
+  type        = string
+  description = "Prefix for account-level resources"
+}
+
+variable image {
+  type        = string
+  description = "Image name"
+}
+
+variable task_role_arn {
+  type        = string
+  description = "ARN for the role assumed by tasks"
+}
+
+variable cmd {
+  type        = list(string)
+  description = "Command to run"
+  default     = []
```

Comment:
the default for the other cmd variable (in `batch/variables`) was `""`, does that matter or will that break things? I noticed all the other default commands were `[]`

Reply:
You're right, we're not using this.